<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>531</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="chartLayout">
     <item>
      <widget class="QGraphicsView" name="graphicsView_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>341</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxYStart"/>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxXStart"/>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxXEnd"/>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxYEnd"/>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEditYEnd">
       <property name="text">
        <string>Y конца:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="lineEditXEnd">
       <property name="text">
        <string>X конца:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEditYStart">
       <property name="text">
        <string>Y начала:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEditXStart">
       <property name="text">
        <string>X начала:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>141</y>
      <width>341</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="text">
        <string>Цвет фона</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="comboBoxClr">
       <item>
        <property name="text">
         <string>Красный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Зеленый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Синий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Черный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Белый</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="text">
        <string>Цвет линии</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxFon">
       <item>
        <property name="text">
         <string>Красный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Зеленый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Синий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Черный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Белый</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="lineEditAlg_2">
       <property name="text">
        <string>Алгоритм</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxAlgos">
       <item>
        <property name="text">
         <string>Бр-хем.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Бр-хем. дейст.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Бр-хем. без ступ.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Алг-м. Ву.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Алг-м. ЦДА.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>280</y>
      <width>341</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Отрисовка пучка</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>310</y>
      <width>341</width>
      <height>192</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEditAlg">
       <property name="text">
        <string>Алгоритм</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEditLen">
       <property name="text">
        <string>Длина отрезка</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="lineEditGrad">
       <property name="text">
        <string>Шаг в град.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBoxAlgos_2">
       <item>
        <property name="text">
         <string>Бр-хем.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Бр-хем. дейст.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Бр-хем. без ступ.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Алг-м. Ву.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Алг-м. ЦДА.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxGrad"/>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxLen"/>
     </item>
     <item row="0" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBoxFon_2">
         <item>
          <property name="text">
           <string>Красный</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Зеленый</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Синий</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Черный</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Белый</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="comboBoxClr_2">
         <item>
          <property name="text">
           <string>Красный</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Зеленый</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Синий</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Черный</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Белый</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="lineEdit_9">
         <property name="text">
          <string>Цвет линии</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_10">
         <property name="text">
          <string>Цвет фона</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>531</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="0">
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonBunch">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>510</y>
      <width>341</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Отрисовать пучок</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClr">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>250</y>
      <width>341</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClr_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>540</y>
      <width>341</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drawHistogram">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>341</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Провести замер</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLine">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>341</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Отрисовать прямую</string>
    </property>
   </widget>
   <zorder>pushButtonClr_2</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>gridLayoutWidget_2</zorder>
   <zorder>lineEdit_8</zorder>
   <zorder>gridLayoutWidget_3</zorder>
   <zorder>gridLayoutWidget_4</zorder>
   <zorder>pushButtonBunch</zorder>
   <zorder>pushButtonClr</zorder>
   <zorder>drawHistogram</zorder>
   <zorder>pushButtonLine</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>923</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
